guides/v2.3/graphql/send-request.md

```diff
@@ -3,7 +3,25 @@ group: graphql
 title: GraphQL requests
 ---
 
-In Magento 2.3.1, you must send all queries and mutations as HTTP POST requests. 
+Magento GraphQL supports the HTTP GET and POST methods. You can send a query as a GET or POST request. GET requests can be cached, but Magento does not support cacheing POST requests. Mutations must be POST requests. 
```

Comment:
Maybe we should remove the mention of caching. It gives the ability to cache, but people might take this to me that Magento does some caching of GET requests.

Reply:
This won't be published on devdocs until 2.3.2. I reworded the passage to 

GET requests can be cached, although Magento does not currently support this feature.

When we support Varnish cacheing, I'll update this passage and include a link to any Varnish documentation that needs to be written in the future.